special-interest-groups/sig-tools/roles-and-organization-management.md

```diff
@@ -0,0 +1,67 @@
+# SIG Tools Roles and Organization Management
+
+## Join and Promotion
+
+### Join
+
+The minimal requirement for contributors to join the Tools SIG is to have
+the **Active Contributor** community role in the [binlog](https://github.com/pingcap/tidb-binlog), [lightning](https://github.com/pingcap/tidb-lightning) and [dm](https://github.com/pingcap/dm) repositories.
```

Comment:
@WangXiangUSTC do we also count `tidb-tools`, and BR / CDC / Dumpling?

Reply:
I think should involve these repo

Reply:
Done